toptal/picasso
# feat(icon): add link icon

### Description

Add Link icon

### How to test

- `yarn start`
- [ensure the Link icon looks good](http://localhost:9001/?path=/story/components-folder--icons)

or check the [temploy](https://picasso.toptal.net/add-link-icon/?path=/story/components-folder--icons)

### Screenshots

![Screen Shot 2020-07-24 at 01 58 02](https://user-images.githubusercontent.com/14301985/88346492-22e99580-cd51-11ea-9537-c4353cb853db.png)

### Review

- [n/a] Annotate all `props` in component with documentation
- [n/a] Create `examples` for component
